performance/videoplayer/build.gradle

```diff
@@ -0,0 +1,68 @@
+def dist = findProperty('org.jetbrains.kotlin.native.home') ?: 'dist'
+
+def linkerOpts = []
```

Comment:
Hmm, this looks like not very awesome idea. Isn't it possible to use composite build with the build in `samples`?

Reply:
Then we have no opportunity 

1) to measure clean way without gradle

2) measure several times

3) use default arguments without -opt, -debug